meilisearch/client.py

```diff
@@ -464,3 +468,75 @@ def wait_for_task(
             An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
         """
         return wait_for_task(self.config, uid, timeout_in_ms, interval_in_ms)
+
+    def generate_tenant_token(
+        self,
+        search_rules: Union[Dict[str, Any], List[str]],
+        *,
+        expires_at: Optional[datetime.datetime] = None,
+        api_key: Optional[str] = None
+    ) -> str:
+        """Generate a JWT token for the use of multitenancy.
+
+        Parameters
+        ----------
+        search_rules:
+            A Dictionary or list of string which contains the rules to be enforced at search time for all or specific
+            accessible indexes for the signing API Key.
+            In the specific case where you do not want to have any restrictions you can also use a list ["*"].
+        expires_at (optional):
```

Comment:
Is there any way to express in the docs that we have a named param? 

CC: @sanders41

Reply:
Not that I have ever seen. I checked around and don't see anything being used in any libraries for this.

However IDEs will show the * in the help text, and also give a too many positional arguments if you try to pass them without a name. Using named params like this has become fairly common in Python so I think it is safe to assume Python developers won't be surprised by this and know what to do.